Harvest.Web/Program.cs

```diff
@@ -19,13 +21,44 @@ public static int Main(string[] args)
 #if DEBUG
             Serilog.Debugging.SelfLog.Enable(msg => Debug.WriteLine(msg));
 #endif
-            // provide some basic console logging (can be captured by Azure logging)
-            Log.Logger = new LoggerConfiguration()
+
+            var isDevelopment = string.Equals(Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT"), "development", StringComparison.OrdinalIgnoreCase);
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(System.IO.Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json")
+                .AddEnvironmentVariables();
+
+            //only add secrets in development
+            if (isDevelopment)
+            {
+                builder.AddUserSecrets<Program>();
+            }
+            var configuration = builder.Build();
+            var loggingSection = configuration.GetSection("Serilog");
+
+            var loggerConfig = new LoggerConfiguration()
                 .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
+                // .MinimumLevel.Override("Microsoft.EntityFrameworkCore", LogEventLevel.Warning) // uncomment this to hide EF core general info logs
                 .MinimumLevel.Override("Microsoft.AspNetCore", LogEventLevel.Warning)
                 .Enrich.FromLogContext()
-                .WriteTo.Console()
-                .CreateLogger();
+                .Enrich.WithClientIp()
+                .Enrich.WithClientAgent()
+                .Enrich.WithExceptionDetails()
+                .Enrich.WithProperty("Application", loggingSection.GetValue<string>("AppName"))
+                .Enrich.WithProperty("AppEnvironment", loggingSection.GetValue<string>("Environment"))
+                .WriteTo.Console();

```

Comment:
Didn't notice this before, but is there a reason we have a `LoggerConfiguration` initialized both here and in SerilogHelpers.cs?

Reply:
@sprucely it's a good question and my guess is no, we can probably dump almost all traces of it from the Startup.cs because we want to be initializing the logs as early as possible and thus Program.cs is the way to go.  But I didn't want to mess with what you did in startup so I left it.  Maybe I'll setup a db and give it a full test with and without before merging